src/login/LoginMain/LoginMain.js

```diff
@@ -32,8 +34,8 @@ class Main extends Component {
   render() {
     return [
       //<FetchingContext.Provider value={this.state}>
-        <SplashContainer key=0 />
-        <Router key=1 history={history}>
+        <SplashContainer key="0" />,
+        <Router key="1" history={history}>
```

Comment:
I would like to mock more UI elements to better follow what is being built and what we need to build next (items with **NEXT** label). Let me know if you have any thoughts.

Personally, I'd suggest: 
- Adding more of the "static" elements shared by all apps, specifically the header and banner (holding the tagline)
- Adding a `<LoginForm />` component (on `path="/login/"`)
    - Adding a "Forgot Password" link to that redirects to the start of the reset password flow (on `path="/reset-password/email"`)
    - **NEXT:** `<LoginForm />` component renders the `<Email />` component
    - **NEXT:** `<LoginForm />` component renders the `<Password />` component 
- Adding a `<ResetPassword />` component at the same level as LoginForm (but on `path="/reset-password/get-reset-email"`)
   - **NEXT:**`<ResetPassword />` component itself renders `<Email />` component 


**The above description above has been added to the code snippet below (apart from the items labelled NEXT)**

```
<SplashContainer key="0" />
<Router key="1" history={history}>
   <div className="dashboard-wrapper">
      <HeaderContainer {...this.props} />
          <div id="dashboard-text">
             <div class="container">
                <h1 class="tagline">eduID is easier and safer login.</h1>
             </div>
             <div id="content">
                <NotificationsContainer />
                <Route
                  exact
                  path="/login/"
                  render={props => <div {...props} ><LoginForm />, has a "forgot password" link that redirects to "/reset-password/get-reset-email"</div>} 
                />
                <Route path="/reset-password/get-reset-email/" component={ResetPassword} />
            </div>
          </div>
        <FooterContainer {...this.props} />
    </div>
</Router>
```